downloaded/installLisk.sh

```diff
@@ -265,7 +265,11 @@ upgrade_lisk() {
 	fi
 
 	echo -e "\\nCopying config.json entries from previous installation"
-	"$LISK_INSTALL"/bin/node "$LISK_INSTALL"/updateConfig.js -o "$LISK_BACKUP"/config.json -n "$LISK_INSTALL"/config.json
+    if [[ -z "${LISK_MASTER_PASSWORD}" ]]; then
```

Comment:
Needs to be made backwards compatible.

Reply:
@fchavant As per my knowledge and @Nazgolze suggested the branch `0.2.0` is only for latest migration. If we need backward compatible to upgrade between 0.9.x versions, we should keep it on seperate branch.

Reply:
Scripts from the `downloaded` directory are made available on downloads.lisk.io without versioning thus I would keep them as backwards compatible as possible.